sdk/communication/azure-communication-networktraversal/src/test/java/com/azure/communication/networktraversal/CommunicationRelayAsyncTests.java

```diff
@@ -0,0 +1,152 @@
+        CommunicationIdentityAsyncClient communicationIdentityClient = setupIdentityAsyncClient(identityBuilder, "createRelayClientUsingManagedIdentity");
+
+        // Action & Assert
+        Mono<CommunicationUserIdentifier> response = communicationIdentityClient.createUser();
+        CommunicationUserIdentifier user = response.block();
+        
+        StepVerifier.create(response)
+        .assertNext(item -> {
+            assertNotNull(item.getId());
+        }).verifyComplete();
+        
+        if (user != null) {
+            Mono<CommunicationRelayConfiguration> relayResponse = asyncClient.getRelayConfiguration(user);
+
+            StepVerifier.create(relayResponse)
+            .assertNext(relayConfig -> {
+                assertNotNull(relayConfig.getIceServers());
+                for (CommunicationIceServer iceS : relayConfig.getIceServers()) {
+                    System.out.println("Urls:" + iceS.getUrls());
+                    assertNotNull(iceS.getUsername());
+                    System.out.println("Username: " + iceS.getUsername());
+                    assertNotNull(iceS.getCredential());
+                    System.out.println("Credential: " + iceS.getCredential());
+                }
+            }).verifyComplete();
+        }
+    }
+
+    @ParameterizedTest
+    @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    public void createRelayClientUsingConnectionString(HttpClient httpClient) {
+        
+        // Arrange
+        CommunicationRelayClientBuilder builder = createClientBuilderUsingConnectionString(httpClient);
+        asyncClient = setupAsyncClient(builder, "createIdentityClientUsingConnectionStringSync");
+        assertNotNull(asyncClient);
+        
+        String connectionString = System.getenv("COMMUNICATION_LIVETEST_DYNAMIC_CONNECTION_STRING");
+        CommunicationIdentityAsyncClient communicationIdentityClient = new CommunicationIdentityClientBuilder()
+            .connectionString(connectionString)
+            .buildAsyncClient();
+        
+        Mono<CommunicationUserIdentifier> response = communicationIdentityClient.createUser();
+        CommunicationUserIdentifier user = response.block();
+        
+        
+        // Action & Assert
+        StepVerifier.create(response)
+        .assertNext(item -> {
+            assertNotNull(item.getId());
+        }).verifyComplete();
+        
+        if (user != null) {
+            Mono<CommunicationRelayConfiguration> relayResponse = asyncClient.getRelayConfiguration(user);
+
+            StepVerifier.create(relayResponse)
+            .assertNext(relayConfig -> {
+                assertNotNull(relayConfig.getIceServers());
+                for (CommunicationIceServer iceS : relayConfig.getIceServers()) {
+                    assertNotNull(iceS.getUrls());
+                    System.out.println("Urls:" + iceS.getUrls());
```

Comment:
We usually don't print things out in tests. Any reason why we need this here?

Reply:
Not really, I'll delete :)